spec/segmenter.html

```diff
@@ -74,11 +77,8 @@ <h1>Internal slots</h1>
       </p>
 
       <p>
-        The value of the [[RelevantExtensionKeys]] internal slot is &laquo; &raquo;.
+        The value of the [[RelevantExtensionKeys]] internal slot is &laquo; *"ss"* &raquo;.
       </p>
-      <emu-note>
-        Intl.Segmenter does not have any relevant extension keys.
-      </emu-note>
```

Comment:
There are actually others: lb, lw, dx. See https://www.unicode.org/reports/tr35/#UnicodeLineBreakStyleIdentifier

Reply:
lb and lw are only for line break, which Intl.Segmenter doesn't support.

dx is not widely used or implemented, and I've raised questions about its utility.